shared/src/packages/projects/libs/validation-schemas/project-search.validation-schema.ts

```diff
@@ -0,0 +1,12 @@
+import joi from 'joi';
+
+import { ProjectValidationMessage } from '../enums/enums.js';
+import { type ProjectSearchParameters } from '../types/types.js';
+
+const projectSearch = joi.object<ProjectSearchParameters, true>({
```

Comment:
```suggestion
const projectFiltering = joi.object<ProjectSearchParameters, true>({
```
this about feature. we will add more field soon